docs/ota-client-guide/modules/ROOT/pages/rotating-signing-keys.adoc

```diff
@@ -7,111 +7,59 @@ We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}
-
-Taking this step is recommended for all production deployments, because a person who gained access to your OTA Connect account would be able to send arbitrary malicious software to your vehicles; that single set of credentials would be a single point of failure. Once you have rotated your keys offline, the maximal impact of an account compromise would be to send an already-signed image--i.e., one that used to be valid.
-
-Before using OTA Connect in production, however, you should create offline keys that you manage yourself, then rotate out the default keys that were automatically created for your account on the OTA Connect server. If you don't do this, you expose yourself to risks that we describe the in the xref:pki.adoc[key management] topic.
-
-Instead of being signed on the server, these metadata files will now be signed locally or on your build machine. The signing happens automatically whenever you push a new disk image to OTA Connect. However, you need to update your build cofiguration first. The following procedures show you how to do this.
-
-Before you start, make sure that you've installed the https://github.com/advancedtelematic/ota-tuf/tree/master/cli[garage-sign] tool.
-
-TIP: `garage-sign` is also packaged with the xref:install-garage-sign-deploy.adoc[`garage-deploy`] packaged releases, so if you're on a Debian-based Linux distro, you can install it that way.
-
-== Rotate the keys for Root and Targets metadata
-
-=== Create a local image repository
-
-An image repository is just a directory structure containing signed metadata in JSON format. Create a new one called _myimagerepo_ with `garage-sign`:
-
+. Create a local image repository.
++
 ----
 garage-sign init --repo myimagerepo --credentials /path/to/credentials.zip
 ----
-
-This command creates a `./tuf/myimagerepo/` directory tree in the current directory.
++
+A `./tuf/myimagerepo/` directory tree is created in the current directory.
 This directory should be secured on an encrypted filesystem.
 
-=== Generate new keys
-
-There are two metadata files in the repo and each file needs a new key to sign it.
-
+. Generate new Root and Targets keys.
++
 ----
 garage-sign key generate --repo myimagerepo --name myroot --type rsa
 garage-sign key generate --repo myimagerepo --name mytargets --type rsa
 ----
++
+IMPORTANT: Keep these keys offline on secure hardware and do not lose them.
```

Comment:
```suggestion
IMPORTANT: Keep these keys offline on secure hardware and do not lose them. If you lose the root key for an environment, it will no longer be possible to update software on any devices connected to that environment. Once you rotate your keys offline, you are responsible for keeping them safe; HERE has no ability to recover them for you.
```